data/campaigns/Northern_Rebirth/scenarios/06a_Old_Friend.cfg

```diff
@@ -299,7 +299,7 @@
 
         [message]
             id=Rakshas
-            message= _ "So! Your forces are managing to advance upon me, are they? Very impressive, but it shall do you no good. KNIGHTS!!"
+            message= _ "So! Your forces are managing to advance upon me? Very impressive, but it shall do you no good. KNIGHTS!!"
```

Comment:
Not a comment on the change, but a comment on this event - I think the entire riders-on-hills ambush should be rewritten, either to give the player hints about the tomato surprise, or to give them a fair retreat.